htdocs/compta/cashcontrol/report.php

```diff
@@ -406,7 +406,7 @@
 		print '<div class="inline-block amount width100"></div>';
 		print '<div class="inline-block amount width100">'.price($cash).'</div>';
 	}
-	if (!$summaryonly && $object->status == $object::STATUS_VALIDATED && $newcash != $object->cash) {
+	if (!$summaryonly && $object->status == $object::STATUS_VALIDATED && price($newcash) != price($object->cash)) {
```

Comment:
price is a text formating function.
For comparison of amount, it should be smantically better to do 
`prcie2num( ) != price2num()`
Can you check this way ?
Of you can also try
`if (float) $newcash != (float) $object->cash`